### PR TITLE
changed colour to rainbow

### PR DESCRIPTION
**Pull request by maplumridge**
because we like rainbows

### CODE REVIEW

File: pattern.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def run():
     # Smooth the result a little
     phi = gaussian_filter(phi, sigma=2.0, mode='wrap')
 
-    plt.imshow(phi, cmap='magma')
+    plt.imshow(phi, cmap='rainbow')
```
Comment (maplumridge) [inline pattern.py:56]: why don't you try ocean next time? more rainbows please.

Comment (andrewmcdonald27) [reply pattern.py:56]: Love this! Bright and colourful and full of joy!

Comment (orlando-code) [reply pattern.py:56]: Asking questions to yourself seems kinda sad...

File: pattern.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def run():
     # Smooth the result a little
     phi = gaussian_filter(phi, sigma=2.0, mode='wrap')
```
Comment (andrewmcdonald27) [inline pattern.py:54]: Try changing sigma to a larger value here to increase the blurriness!

**Review by andrewmcdonald27 — CHANGES_REQUESTED** [review]

**Review by orlando-code — COMMENTED** [review]
all good

**Comment by maplumridge** [comment]
please don't close this

**Review by orlando-code — APPROVED** [review]

**Pull request merged**